src/core.c

```diff
@@ -578,6 +580,46 @@ void PrintHelp() {
     printf_ftrace(0, " options are:\n");
     printf_ftrace(0, "    '-v'|'--version' to print box64 version and quit\n");
     printf_ftrace(0, "    '-h'|'--help' to print this and quit\n");
+    printf_ftrace(0, "    '-k'|'--kill-all' to kill all box64 instances\n");
+}
+
+void KillAllInstances()
+{
+
+    struct dirent* entry;
+    ssize_t len;
+    char proc_path[PATH_MAX];
+    char exe_path[PATH_MAX];
+    char exe_target[PATH_MAX];
+
+    DIR* proc_dir = opendir("/proc");
```

Comment:
At the end of this function, you should release `/proc` directory stream. That is, `closedir(proc_dir)`.

Reply:
Thanks, fixed. Also changed the process name from hard-coded `"box64"` to the name of self process.